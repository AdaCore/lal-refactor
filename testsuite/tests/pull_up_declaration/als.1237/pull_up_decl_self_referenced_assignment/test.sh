pull_up_declaration -P default.gpr --source main.adb --line 9 --column 19
