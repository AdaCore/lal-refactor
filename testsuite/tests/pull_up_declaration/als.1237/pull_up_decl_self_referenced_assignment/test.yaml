title: pull_up_decl_self_referenced_assignment

description: pull_up_declaration tool test

driver: shell_script
