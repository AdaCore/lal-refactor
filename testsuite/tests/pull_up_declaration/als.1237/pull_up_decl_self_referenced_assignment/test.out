main.adb
4:1-4:1    procedure Extracted (Number : in out Integer);

   procedure Extracted (Number : in out Integer) is
   begin
      for J in 1 .. 10 loop
         Number := Number + 1;
      end loop;
   end Extracted;


7:1-15:1 
17:16-17:16  (Number => Number)

